app/src/main/java/org/kinecosystem/kinit/view/comingSoon/EcoAppsComingSoonWebFragment.kt

```diff
@@ -0,0 +1,53 @@
+package org.kinecosystem.kinit.view.comingSoon
+
+import android.annotation.SuppressLint
+import android.databinding.DataBindingUtil
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.webkit.WebSettings
+import android.webkit.WebView
+import android.webkit.WebViewClient
+import org.kinecosystem.kinit.BuildConfig
+import org.kinecosystem.kinit.R
+import org.kinecosystem.kinit.databinding.ComingSoonWebLayoutBinding
+import org.kinecosystem.kinit.view.BaseFragment
+import org.kinecosystem.kinit.viewmodel.EcoAppsComingSoonViewModel
+
+
+class EcoAppsComingSoonWebFragment : BaseFragment(){
+
+    companion object {
+        fun getInstance(): EcoAppsComingSoonWebFragment {
+            return EcoAppsComingSoonWebFragment()
+        }
+    }
+
+    lateinit var model: EcoAppsComingSoonViewModel
+    lateinit var binding: ComingSoonWebLayoutBinding
+
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
+        binding = DataBindingUtil.inflate(inflater, R.layout.coming_soon_web_layout, container, false)
+        model = (activity as EcoAppsComingSoonActivity).getModel()
```

Comment:
why do you create the model in the activity and pass it 
and not create it in the fragment

Reply:
no better work with fragments as i see it